source/modules.tex

```diff
@@ -495,11 +495,12 @@
 exported \grammarterm{module-import-declaration}{s}
 in $T$; such translation units are
 said to be \defnx{exported}{module!exported} by $T$.
-When a \grammarterm{module-import-declaration} in a module unit imports
-another module unit of the same module, it also imports
-all translation units imported by
-all \grammarterm{module-import-declaration}{s}
-in that module unit.
+Additionally, when a \grammarterm{module-import-declaration}
+in a module unit of some module $M$ imports another module unit $U$ of $M$,
```

Comment:
Should we be using \placeholder here instead of math font?

Reply:
We have a healthy variety of opinions from the history of editorial meetings here: #326, #3139, #3271.
The latest and greatest is to use math variables for everything, so that's what I'm doing here.

Reply:
Would be good to get some examples of when to use what added to our guidelines.